mithril-aggregator/src/services/prover.rs

```diff
@@ -410,6 +412,50 @@ mod tests {
         transactions_set_proof[0].verify().unwrap();
     }
 
+    #[tokio::test]
+    async fn cant_compute_proof_for_not_yet_certified_transaction() {
+        let total_block_ranges = 5;
```

Comment:
Is it useful to declare a variable here ?  
Builder function names seem to gave enough information about the value.

Reply:
Actually, all the tests in the module are structured this way. 
But I agree we can skip the variables as the function names are clear enough.

Reply:
Done